ldes-server-port-ingestion-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/ingestion/rest/LdesMemberIngestionController.java

```diff
@@ -10,14 +11,19 @@
 public class LdesMemberIngestionController {
 
 	private final MemberIngestService memberIngestService;
+	private final LdesShaclValidator ldesShaclValidator;
 
-	public LdesMemberIngestionController(final MemberIngestService memberIngestService) {
+	public LdesMemberIngestionController(final MemberIngestService memberIngestService,
```

Comment:
Je mag het zo laten maar het voordeel van valideren adhv annotatie zou inhouden dat we niet in alle controllers deze validator moeten injecteren met bijhorende if statements en eigenlijk enkel een paar @Valid zouden moeten toevoegen.

Reply:
juist! moest ik nog voor kijken!